### DISCUSSION TITLE
PWA's with Next js....

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#39567] [Help] [answered]
May I get some feedback please???

I have a website that is built with Next.js...  I would like to transform it into a Progressive Web App... of course I know NOTHING about PWA's.....      but sure I would like to learn...

Are there any references on how to get a Next.js site and make it a PWA???

**Comment by gaston-flores** [comment]
You should have a look at [next-pwa](https://www.npmjs.com/package/next-pwa), there's very little configuration required. You don't need anything specific to Next to make a PWA (basically just a web manifest and a service worker) but this is the easiest way.

**Comment by leerob** [comment, marked as answer]
Hey everyone. We now have updated documentation on building PWAs with Next.js.

https://nextjs.org/docs/app/building-your-application/configuring/progressive-web-apps

**Comment by Emmaccen** [comment]
> May I get some feedback please???
> 
> I have a website that is built with Next.js... I would like to transform it into a Progressive Web App... of course I know NOTHING about PWA's..... but sure I would like to learn...
> 
> Are there any references on how to get a Next.js site and make it a PWA???

Hi, Elindo. Not sure if you later went through with learning about PWAs but if you do need a full documentation on everything PWA, do check out https://web.dev/learn/pwa.

Cheers 🥂
